content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -247,20 +262,85 @@ zones, or regions.
     caption="A service mesh after a control plane instance fails"
     >}}
 
-The following list ranks control plane deployment examples by availability:
+### Independent Control Planes
+
+You can also deploy a control plane per cluster, with a shared root CA. Each cluster's control plane
```

Comment:
I'd like to reword this a tiny bit to make it more part of a continuum, so rather than "also"something like:

For maximum availability and configuration isolation you can deploy a control plane in each cluster.

We don't need to mention shared root CA since that is the same for shared control planes and independent control planes.

Reply:
+1, the key is config isolation.  Perhaps max availability as well.  (especially if we remove shared control plane with failover).